File: DoublyLinkedListDart/doublylist/lib/structures/node.dart
class Node <T>{

  T _data;
  Node<T>? _next; 
  Node<T>? _previous;

  Node(this._data);

  T get data => _data;

  set data(T value) {
    _data = value;
  }

  Node<T>? get next => _next;

  set next(Node<T>? value) {
    _next = value;
  }

  Node<T>? get previous => _previous;

  set previous(Node<T>? value){
    _previous = value;
  }
  @override
  String toString(){
    return "$_data";
  }

}

File: Graphs/AdyacenceMatrix/src/co/edu/uptc/structures/linear/Node.java
package co.edu.uptc.structures.linear;

public class Node<T> {
    private T data;
    private Node<T> next;
    private Node<T> previous;

    public Node(T data) {
        this.data = data;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }

    public Node<T> getNext() {
        return next;
    }

    public void setNext(Node<T> next) {
        this.next = next;
    }

    public Node<T> getPrevious() {
        return previous;
    }

    public void setPrevious(Node<T> previous) {
        this.previous = previous;
    }

    @Override
    public String toString() {
        return "Node{" +
                "data=" + data +
                ", next=" + next +
                '}';
    }
}